CONTRIBUTING.rst

```diff
@@ -95,6 +95,10 @@ To generate a local version:
     ./build.sh
     ./start_doc_server.sh
 
+.. note::
+    The docs build script ``build.sh`` requires bash 4.0 or greater.
```

Comment:
@dstandish: Or maybe we should tell people to use `breeze` instead? (we can wait with that until I merge some simplifications and optimisations to make it quite a bit lighter). 

Breeze has stabilised a lot and it is being used by more people now so I think that might be good time to make it the "default" dev environment.

We have `./breeze --build-docs` command that runs the build in automatically downloaded/updated docker container so no matter what  OS you run it all it will yield the same results (note that it is written in the way that it will generate the output files in host (the volumes are mounted from host during build). And you do not need the doc server in fact as you can simply open the index.html locally and it will work the same (except search maybe).

Some more context: I have plans to remove all the locally run scripts from the documentation and leave breeze as the default way of doing most development tasks. I am going to remove most of the docs referring to the scripts and leave breeze docs only.

WDYT @dstandish ? We can add this comment now, but maybe going 'breeze-first' should be the next step ?

Reply:
yeah i totally get the value of using breeze for everything ... and that's what i used initially... just when i noticed that there was a host script it seemed significantly faster than docker so that's what i started using.  i try to stay in virtualenv when i can.

but yesterday when i moved to a new machine, i forgot that i needed to upgrade bash again and stumbled around for a while (googling how to install mapfile etc).  so that was the impetus for adding.

but i defer to you on this completely -- whatever you think is best.  if you are getting rid of host build script soon then yeah probably makes makes sense not to bother.

thanks

Reply:
It's perfectly OK to merge it now. I just wanted to hear your opinion on breeze's usage in this case. I will make some changes soon to make Breeze faster/more appealing and then I will remove all the reference to scripts :)